ui/src/shared/components/cells/Cell.tsx

```diff
@@ -66,8 +93,12 @@ class CellComponent extends Component<Props> {
             onCSVDownload={this.handleCSVDownload}
           />
         )}
-        <div className="cell--view" data-testid="cell--view-empty">
-          {this.view}
+        <div
+          className="cell--view"
+          data-testid="cell--view-empty"
+          ref={this.cellRef}
+        >
+          {inView && this.view}
```

Comment:
instead of returning null here when inView===false, should we be rendering Spinner Container? this.view returns spinnercontainer if viewsstatus is done, maybe viewsstatus could be loading if inView is false.